ironic-images/ipa-debian-bookworm/ipa-debian-bookworm.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# elements path - need to include both the ironic-python-agent-builder DIB packages
+# and our custom_elements packages
+# export ELEMENTS_PATH=/root/keekz-image-build/venv/share/ironic-python-agent-builder/dib:/root/keekz-image-build/ipa-build-20240801/custom_elements
+
+# distro version
+export DIB_RELEASE=bookworm
+
+# devuser element - https://github.com/openstack/diskimage-builder/tree/master/diskimage_builder/elements/devuser
+export DIB_DEV_USER_USERNAME=undercloud
+export DIB_DEV_USER_PASSWORD=secret_password
+export DIB_DEV_USER_PWDLESS_SUDO=yes
+
+# https://docs.openstack.org/diskimage-builder/latest/elements/simple-init/README.html#fallback
+export DIB_SIMPLE_INIT_NO_DHCP_FALLBACK=1
```

Comment:
This is ignored by the IPA DIB so it has no effect so it can be removed. https://opendev.org/openstack/ironic-python-agent-builder/src/branch/master/dib/ironic-python-agent-ramdisk/element-deps pulls in `dhcp-all-interfaces` which will override this.